dissect/target/plugins/child/virtuozzo.py

```diff
@@ -0,0 +1,38 @@
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers.record import ChildTargetRecord
+from dissect.target.plugin import ChildTargetPlugin
+
+
+class VirtuozzoTargetPlugin(ChildTargetPlugin):
+    """Child target plugin that yields from Virtuozzo container's root.
+
+    Virtuozzo conatiners are by default registered in the folder ``vz/root/$VEID``,
+    where VEID will be substituted with the actual container UUID.
+
+    /
+      etc/
+      var/
+      vz/
+          root/
+              <container-uuid>/
+              <container-uuid>/
+
+    Sources:
+        - https://docs.virtuozzo.com/virtuozzo_hybrid_server_7_command_line_reference/managing-system/configuration-files.html
+    """
```

Comment:
```suggestion
    """  # noqa: E501
```
This will fix the linting error.